packages/common/container-definitions/src/runtime.ts

```diff
@@ -59,6 +59,7 @@ export enum AttachState {
 export interface IRuntime extends IDisposable {
 	/**
 	 * Executes a request against the runtime
+	 * @deprecated - TODO: will be removed from interface in future major release
```

Comment:
We should add some info about the alternative, i.e., using entryPoints instead. Maybe once we have made progress on the document that Mark is working on, we can link to it.

Reply:
Same goes for other places this is deprecated.